html/themes/custom/common_design_subtheme/sass/pocam_custom/_home.scss

```diff
@@ -31,13 +31,13 @@
     @include font-size($cd-font-size-medium);
     font-family: $cd-font;
     text-align: left;
-    padding: 22px 10px;
+    padding: 0 10px;
     border: 1px solid $cd-ocha-blue;
     color: $cd-ocha-blue;
     transition: .3s ease;
     cursor: pointer;
     height: 42px;
-    width: 100%;
+    width: 98%;
```

Comment:
@lazysoundsystem I've been using a media query for IE11 fixes sometimes (depends on the context)
Eg. https://github.com/UN-OCHA/common_design/blob/d08c5d4faf3fda1f4b73481eac4f0bbd034f0da9/sass/cd/cd-header/_cd-user-menu.scss#L35
This is handy so as not to affect the styles in all other browsers. Might not be applicable in this particular case, so just an FYI

Reply:
Thanks @left23 - I was going to ask you for a review but knew you're busy with other things...

I don't think it effects anything else, but it's useful to mark it as an ie fix with the media query. 

And thanks for dealing with ie issues in the common design - I just added the class and it all works fine, without having to dig for what was causing the issue.